Makefile

```diff
@@ -112,8 +114,10 @@ depend.update: ${DEP} ; $(info $(H) ensuring dependencies are up to date...)
 	${DEP} ensure -update
 	cp Gopkg.lock vendor/Gopkg.lock
 
+# If CGO_ENABLED=0 then go get tries to install in system directories.
+# If -pkgdir <dir> is also used then various additional .a files are present.
 ${DEP}:
-	unset GOOS && go get -u github.com/golang/dep/cmd/dep
+	unset GOOS && CGO_ENABLED=1 go get -u github.com/golang/dep/cmd/dep
```

Comment:
This is no longer needed ?

Reply:
I'm not sure I understand the question.  I originally did add this as part of evaluating tradeoffs (especially after I tried to fix the spelling of the "export CGO_ENABLE=0"), but as I mentioned in the original PR comment my reason for adding it was to help insulate the build from non-standard parameters that might be set in the outside environment.  If this is too much of a corner case then I can remove it.  I suppose if I was really worried about this then I should be setting "export CGO_ENABLED=1" in the Makefile rather than leaving it unmodified.